adal/src/main/java/com/microsoft/aad/adal/Oauth2.java

```diff
@@ -27,6 +27,7 @@
 import android.os.Build;
 import android.text.TextUtils;
 import android.util.Base64;
+import android.util.Log;
```

Comment:
I will change it to Logger. BTW, why are we using log (i.e. AuthenticationActivity.java) sometimes while using logger at the same time?